sys/include/net/gnrc/netif.h

```diff
@@ -478,6 +490,12 @@ char *gnrc_netif_addr_to_str(const uint8_t *addr, size_t addr_len, char *out);
  */
 size_t gnrc_netif_addr_from_str(const char *str, uint8_t *out);
 
+/**
+ * @brief   gnrc netif thread event handler.
+ *
+ * @param[in] args   The network interface
+ */
+void *gnrc_netif_thread(void *args);
```

Comment:
Mhhh... Can we somehow get around exposing this function?

Reply:
Maybe you can use a trampoline function somehow?